CONTRIBUTING.adoc

```diff
@@ -0,0 +1,36 @@
+= Contributing to Eclipse uProtocol
```

Comment:
@sophokles73 reminded me we wanted to link to https://github.com/eclipse-uprotocol/.github/blob/main/CONTRIBUTING.adoc instead of re-writing this in each repo

Reply:
Good point. I'll update it.

Reply:
Done.